### PR TITLE
Improve queue creation and job enqueueing docs

### PR DESCRIPTION
**Pull request by KyleGoyette**
## Description
What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

Adds a gif showing updating the queue with gpu info:
image

Modifies the launching a job section to include info about modifying the resource config

## Ticket
Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist
Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply. 

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/launch/getting-started.md [hunk 1]
```diff
@@ -179,12 +180,13 @@ When you click the button, a drawer will slide from the right side of your scree
...
 * **Resource**: the execution platform for jobs in this queue. Check out the other options, but for this walkthrough we will use the default: **Docker container**.
-* **Configuration**: json configuration that will passed to any launch agent that polls on this queue. This can be left blank, but some additional configuration does need to be provided in order to leverage GPU.
+* **Configuration**: default resource configurations for all jobs placed on this queue. Used to specify arguments for the resource when run. These defaults can be overridden when you enqueue a job.
 
 ![](/images/launch/create-queue.gif)
 
 :::info
-Add the following resource configuration in order to use GPUs in jobs submitted to this queue:
+
+Accepted configuration values include all arguments available for the `docker run` command. For more information, see the [reference](https://docs.docker.com/engine/reference/commandline/run). Add the following resource configuration in order to use GPUs in jobs submitted to this queue:
```
Comment (KyleGoyette) [inline docs/guides/launch/getting-started.md]: Need to explain how to support lists of args

File: docs/guides/launch/getting-started.md [hunk 2]
```diff
@@ -179,11 +180,23 @@ When you click the button, a drawer will slide from the right side of your scree
...
+
+```json
+{
+    "env": [
+        "MY_ENV_VAR=value",
+        "MY_EXISTING_ENV_VAR"
+    ]
+}
+```
+
```
Comment (bcsherma) [inline docs/guides/launch/getting-started.md]: Can we add one more example here? Like:

```json
{
    "env": [
        "MY_ENV_VAR=value",
        "MY_EXISTING_ENV_VAR"
    ],
    "volume": [
         "/mnt/datasets:/mnt/datasets"
     ]
}
```

File: static/images/launch/launch-job-update-resource.gif
Comment (bcsherma) [inline static/images/launch/launch-job-update-resource.gif:1]: I think you should re-record this with the content enlarged

**Review by bcsherma — APPROVED** [review]
overall fire changes, a couple little thingies and one onerous request

File: static/images/launch/launch-job-update-resource.gif
Comment (KyleGoyette) [reply static/images/launch/launch-job-update-resource.gif:1]: This is indeed an ONEROUS request

Comment (KyleGoyette) [reply static/images/launch/launch-job-update-resource.gif:1]: All right, updated.

**Pull request merged**